### DISCUSSION TITLE
Issues with AIO migration

### DISCUSSION THREAD

**Discussion by Deezer87** [nextcloud/all-in-one#4730] [Questions] [answered]
Trying to migrate from Ubuntu VM running 28.0.5.1 to the AIO docker. I have a couple of issues:
1) when I make changes to config.php using sudo docker run -it --rm --volume nextcloud_aio_nextcloud:/var/www/html:rw alpine sh -c "apk add --no-cache vim && vi /var/www/html/config/config.php", the changes are not persistent through reboot. How do I make persistent changes to the config.php file?

2) I had folders shared that when synced would simply sync as as the folder name. Now it is trying to sync as "Shared/Folder". Is there a setting or work around for this?

**Comment by szaimen** [comment, marked as answer]
Hi, 

> 1. when I make changes to config.php using sudo docker run -it --rm --volume nextcloud_aio_nextcloud:/var/www/html:rw alpine sh -c "apk add --no-cache vim && vi /var/www/html/config/config.php", the changes are not persistent through reboot. How do I make persistent changes to the config.php file?

What kind of changes are not persistet through a reboot?

> 2\. I had folders shared that when synced would simply sync as as the folder name. Now it is trying to sync as "Shared/Folder". Is there a setting or work around for this?

Yes, it is this value: https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#share-folder

**Reply by Deezer87** [reply]
The 'trusted_proxies'. It changes the IP address I add to '::1'. So I have 2 '::1' entries.
